### DISCUSSION TITLE
Can edit labels, assigned user etc. on repos you dont have access to do so

### DISCUSSION THREAD

**Discussion by iGabyTM** [community/community#14164] [Mobile]
You can view the `Edit` button for these options on all repos. The labels are temp added but after a page refresh they are gone.

App version `1.59.0` (using beta)
OS version`Android 10`
Phone `Samsung A51`

https://github.com/oraxen/oraxen/pull/306
![Screenshot_20220403-121145_GitHub.png](https://user-images.githubusercontent.com/32105478/161420434-13610134-1ce4-48a3-a108-7789af28ea69.png)

**Comment by iskae** [comment]
Hi @iGabyTM 👋 

Thanks for your feedback! We have been trying to reproduce the issue but unfortunately we were not successful so far 😅  Would you be able to help us further?

- As far as I can see you are a contributor of the repository that was mentioned. Are you able to change those fields via web?
- Is this behaviour happening for all sections or only for `Labels`?
- Do you receive any error messages after updating any option?

Thanks so much in advance!

**Reply by thispsj** [reply]
@iskae I've been able to reproduce this but on Android 6.0. GitHub doesn't support Android 6.0 now so ig it won't fix for me.

I'll try and update for Android 7.0+

**Comment by iGabyTM** [comment]
The problem was fixed on an update, this discussion can be closed!
